src/ImageSharp/Configuration.cs

```diff
@@ -125,7 +125,7 @@ public void Configure(IConfigurationModule configuration)
         /// Creates a shallow copy of the <see cref="Configuration"/>
         /// </summary>
         /// <returns>A new configuration instance</returns>
-        public Configuration ShallowCopy()
+        public Configuration Clone()

```

Comment:
I think this is misleading now. `IDeepCloneable.Clone()` does a deep copy + we also need to make `Configuration` deep clonable.